strawberryfields/ops.py

```diff
@@ -1411,6 +1413,29 @@ def _apply(self, reg, backend, **kwargs):
         return ancilla_val / s
 
 
+class PassiveChannel(Channel):
+    r"""Perform an arbitrary multimode passive operation
+
+    Args:
+        T (array): an NxN matrix acting on a N mode state
+
+    .. details::
+
+        Acts the following transformation on the state:
+
+        .. math::
+            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}j
+    """
+
+    def __init__(self, T):
+        super().__init__([T])
+        self.ns = T.shape[0]
+
+    def _apply(self, reg, backend, **kwargs):
+        p = par_evaluate(self.p)
+        backend.passive(p[0], *reg)
+
+
```

Comment:
since the Fock backends do not know how to deal with `passive` operations, we should add a decompositions method by using the SVD of the passive operation. Similarly, it should be fairly straightforward to add support to the bosonic backend. I get that this can be added in a future PR... If that is the case, are we raising the correct `Not Implemented error` when this function is called?

Reply:
💯